content/docs/start/experiments.md

```diff
@@ -25,6 +25,11 @@ $ dvc run -n evaluate \
 
 ### 💡 Expand to see what happens under the hood.
 
+The `-M` flag here specifies a metrics file, while the `--plots-no-cache` flag
+specifies a plot metrics file produced by this stage that will not tracked by
+DVC. You can choose whether to ignore the files in DVC or cache them. See
```

Comment:
`You can choose whether to ignore the files in DVC or cache them.` - it feels repetitive again with the previous sentence.

Reply:
Oh, I thought that without the latter statement, it just wasn't clear to the user that there is an option to cache them too.
Should I remove this last statement, nevertheless?

Reply:
Yes, please remove the last 2 sentences. I summarized everythin this paragraph needs in https://github.com/iterative/dvc.org/pull/1511#discussion_r448744635 above. Resolving here.